### DISCUSSION TITLE
Modify replay speed for live timings

### DISCUSSION THREAD

**Discussion by fkarg** [f1multiviewer/issue-tracker#519] [Feature requests]
In all video streams, you can adjust the replay speed (not live) in the settings in the bottom right. This setting does not sync across multiple running video streams with `s` (fair enough, but would be a nice QoL).

However, as far as I found, it is not possible to increase replay speed for anything related to live timing replays. This includes the main 'Replay Live Timing' window and all additional windows synced to timestamps but fixed to 1x replay speed: Track Map, Track Limits Tracker, Investigations Tracker, Team Radio, etc.

It is possible to 'jump forward' by varying amounts of time, so I similarly expected a way to increase replay speed, but this does not appear to exist.

While I'm mostly interested in speeding it up (I'm watching past races at 1.5x), there's probably a similar a use-case for the available slower speeds.

In fact, this has been reported by other users as a bug in the past, see:
- https://github.com/f1multiviewer/issue-tracker/issues/424
- https://github.com/f1multiviewer/issue-tracker/issues/460
- https://github.com/f1multiviewer/issue-tracker/issues/469

**Comment by AnderzL7** [comment]
+1

If I can’t catch a session live, I usually watch the replay at 1.5x or 2x speed. I’d like to have Live Timing open on my side monitor while doing so, but since it's locked to 1x speed, it gradually falls out of sync with the replay.
